### PR TITLE
Add project name and domain name as args for functional tests

### PR DESCRIPTION
**Pull request by squiishyy**

### CODE REVIEW

**Review by katrogan — APPROVED** [review]

File: boilerplate/flyte/end2end/run-tests.py [hunk 1]
```diff
@@ -258,6 +260,18 @@ def run(
     is_flag=True,
     help="Abort failing workflows upon exit",
 )
+@click.option(
+    "--test_project_name",
+    default="flytesnacks",
+    is_flag=False,
```
Comment (katrogan) [inline boilerplate/flyte/end2end/run-tests.py:266]: is this not the default?

File: boilerplate/flyte/end2end/run-tests.py [hunk 2]
```diff
@@ -258,6 +260,18 @@ def run(
     is_flag=True,
     help="Abort failing workflows upon exit",
 )
+@click.option(
+    "--test_project_name",
+    default="flytesnacks",
```
Comment (katrogan) [inline boilerplate/flyte/end2end/run-tests.py:265]: can we also add
`type=str,`
here?

File: boilerplate/flyte/end2end/run-tests.py (hunk 1 above)
Comment (squiishyy) [reply boilerplate/flyte/end2end/run-tests.py:266]: `flytesnacks` is the default yeah

**Comment by squiishyy** [comment]
@katrogan sorry!! I did not mean to "dismiss a stale review". Not even sure how i did that lol. I added those changes though!

File: boilerplate/flyte/end2end/run-tests.py (hunk 1 above)
Comment (katrogan) [reply boilerplate/flyte/end2end/run-tests.py:267]: oh sorry, I mean is `is_flag=False` already the default?
but no problem being explicit!

**Review by katrogan — APPROVED** [review]

**Pull request merged**